webpack.config.babel.js

```diff
@@ -0,0 +1,47 @@
+import path from 'path';
+import DashboardPlugin from 'webpack-dashboard/plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+
+
+export default {
+    entry: './src/js/index.js',
+    output: {
+        filename: 'bundle.js',
+        path: path.resolve(__dirname, 'dist/build')
+    },
+    resolve: {
+        alias: {
+            '~': path.resolve(__dirname, 'src/js')
+        }
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules|bower_components)/,
+                use: [
+                    {
+                        loader: 'babel-loader',
+                        options: {
+                            presets: ['env', 'react']
+                        }
+                    },
+                    {
```

Comment:
You can comment out the eslint if its causing issues. I was just trying it out XD